app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
```

Comment:
- [ ] Kindly note that the comments table will fail to locate the user's table because the foreign key is `author_id`, you need to indicate that the class name(model) is `User`, like so: `belongs_to :author, foreign_key: 'author_id', class_name: "User"`